gax-java/gax-httpjson/src/main/java/com/google/api/gax/httpjson/InstantiatingHttpJsonChannelProvider.java

```diff
@@ -124,6 +127,11 @@ public TransportChannelProvider withEndpoint(String endpoint) {
     return toBuilder().setEndpoint(endpoint).build();
   }
 
+  @Override
+  public TransportChannelProvider withUseS2A(boolean useS2A) {
+    return toBuilder().setUseS2A(useS2A).build();
```

Comment:
I know we are going to support S2A in HttpJson transport in the future, but I think it is better to leave this as a no-op rather than actually setting `useS2A`, since `useS2A` is not used at this point for `InstantiatingHttpJsonChannelProvider`.